plugin/discovery/meta.go

```diff
@@ -0,0 +1,28 @@
+package discovery
+
+import (
+	"github.com/blang/semver"
```

Comment:
We already use it transitively, so go with it for now if it does the job. They looks pretty similar, so switching wouldn't be too bad if we do find some issue in the future.